.github/travis/check_license.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
Can you update this with the latest version from https://github.com/SymbiFlow/python-symbiflow-v2x/blob/master/.github/ci/check_license.sh ?

Reply:
Done